.github/workflows/benchmark.yml

```diff
@@ -0,0 +1,29 @@
+on:
+  push:
+    branches:
+      - master
+      - main
+    paths:
+      - "**"
+      - "!**.md"
+  pull_request:
+    paths:
+      - "**"
+      - "!**.md"
+
+name: Run Benchmarks
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+    steps:
+      - name: checkout
+        uses: actions/checkout@v2
+      - name: gobenchdata publish
+        uses: bobheadxi/gobenchdata@v1
+        with:
+          PRUNE_COUNT: 30
+          PUBLISH: true
+          CHECKS: true
+          PUBLISH_BRANCH: gh-pages
+        env:
+          GITHUB_TOKEN: ${{ secrets.ACCESS_TOKEN }}
```

Comment:
It this a PAT token?

Reply:
I forgot modifying here. It's better to use `secrets.GITHUB_TOKEN`